packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -133,10 +146,7 @@ export function TransferPanel() {
 
   const isCctpTransfer = useIsCctpTransfer()
 
-  const latestEth = useLatest(eth)
-
-  const isConnectedToArbitrum = useLatest(useIsConnectedToArbitrum())
```

Comment:
`useIsConnectedToArbitrum` and `useIsConnectedToOrbitChain` aren't used anymore, and I don't think there will be much use for them. I'd just kill this code